modules/core/src/main/scala/org/tessellation/Main.scala

```diff
@@ -32,7 +33,7 @@ object Main
 
   val opts: Opts[Run] = cli.method.opts
 
-  val kryoRegistrar: Map[Class[_], Int] = coreKryoRegistrar ++ dagSharedKryoRegistrar
+  val kryoRegistrar: Map[Class[_], Int Refined _] = coreKryoRegistrar ++ dagSharedKryoRegistrar
```

Comment:
What about:
```
Int Refined Greater[100]
```
or something like that? In fact, there is no upper bound but there is a lower bound 100 or 200 (don't remember exactly, needs to be verified with ADR)

Reply:
In this case there will be type mismatch like below: <scala.collection.immutable.Map[java.lang.Class[_$1], eu.timepit.refined.api.Refined[scala.Int, eu.timepit.refined.boolean.And[eu.timepit.refined.boolean.Not[eu.timepit.refined.numeric.Less[Int(600)]], eu.timepit.refined.boolean.Not[eu.timepit.refined.numeric.Greater[Int(699)]]]]]|scala.collection.IterableOnce[(java.lang.Class[_$1],
eu.timepit.refined.api.Refined[scala.Int, eu.timepit.refined.numeric.Greater[100]])]>

I will try suggestion from @kpudlik .